soc/arm/ti_k3/j721e/Kconfig.defconfig.series

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_TI_J721E
+
+source "soc/arm/ti_k3/j721e/Kconfig.defconfig.j721e*"
+
+config SOC_SERIES
+	default "j721e"
+
+config NUM_IRQS
+	default 512
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 25000000
+
```

Comment:
Can you help us understand how you came up with these values?

Reply:
I took it from TI PDK.

Reply:
Okay thanks for clarifying, this will make sense here when DM timer support is added.